standard/classes.md

```diff
@@ -1614,8 +1614,10 @@ member_name
 
 method_body
     : block
+    | '=>' expression ';'
```

Comment:
I'm really confused now. The semi-colon on the next line is just the "end of rule" indicator, yes? So do we actually need another line of:

```text
| ';'
```

so that a semicolon counts as a body on its own? (We describe that as being valid at line 1642.)

Reply:
(And to be clear, I suspect this was a bug before this PR, so I'm happy to punt it to a separate issue.)

Reply:
(I note that operator_body *does* have a `| ';'` option btw.)

Reply:
I think we *do* need the `|  ;` addition.  I made that change.